backend/app/notify/notify.go

```diff
@@ -35,13 +35,15 @@ type Destination interface {
 type Store interface {
 	Get(locator store.Locator, id string, user store.User) (store.Comment, error)
 	GetUserEmail(siteID string, userID string) (string, error)
+	GetUserTelegram(siteID string, userID string) (string, error)
 }
 
 // Request notification for a Comment
 type Request struct {
-	Comment     store.Comment
-	parent      store.Comment
-	Emails      []string
+	Comment       store.Comment
+	parent        store.Comment
+	Emails        []string
+	TelegramUsers []string
```

Comment:
is it really users? i.e. can this be a group/channel id? Maybe call it "Telergams"?

Reply:
Renamed to the Telegrams, I agree with your point. I think it's technically possible to add user notifications to the channel (however, it would be quite a strange idea).